scripts/release_prep.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+set -x
+
+# pre-requisites: install github CLI
+# - github documentation: https://github.com/cli/cli#installation
+# - github is remote 'origin'
+# - PRs use test section LAST with heading "## Tests"
+# - ALL build and release PRs start with "build: "
+
+git fetch --all --tags
+git reset --hard
```

Comment:
this line is gonna suck if we have changes we forgot about... should we remove this line and just let the script error if the user has unstaged changes